open-metadata-implementation/access-services/analytics-modeling/analytics-modeling-server/src/main/java/org/odpi/openmetadata/accessservices/analyticsmodeling/synchronization/IdentifierResolver.java

```diff
@@ -218,10 +216,10 @@ private void resolveAssetReferences() {
 	 * @param methodName for logging.
 	 * @return list of elements
 	 */
-	private List<EntityDetail> getSchemaAttributes(String assetQualifiedName, String methodName) {
+	public List<EntityDetail> getSchemaAttributes(String assetQualifiedName, String methodName) {
 		
 		List<EntityDetail> metadata = new ArrayList<>();
-		String pattern = QualifiedNameUtils.escapeQualifiedNameRegExPattern(assetQualifiedName) + ANY_STRING_REGEX;
+		String pattern = QualifiedNameUtils.escapeQualifiedNameRegExPattern(assetQualifiedName) + ANY_QNAME_REGEX;
```

Comment:
This function is available in the reposiotry helper

Reply:
changed to getStartsWithRegex(...)